Assets/cannonshot.cs

```diff
@@ -5,14 +5,28 @@
 public class cannonshot : MonoBehaviour
 {
     // Start is called before the first frame update
-    void Start()
-    {
-        
-    }
 
+    public Transform cannonPoint;
+    public GameObject cannonPrefab;
+
+    public float cannonForce = 12f;
+    public float firerate;
+    float nextfire;
```

Comment:
would 
```suggestion
    private float nextfire;
```
work as well?